app/controllers/preview_controller.rb

```diff
@@ -1,90 +1,77 @@
 # returns preview (thumbnail), large, and full-size JPEG images
```

Comment:
This file needs some `TODO` documentation added:
* `return_image_datastream` depends on as-yet-nonexistent `SolrService`
* `IIIF_SERVER` is not currently set in `.env`
* `datastream_disseminator_url` should not depend on `ActiveFedora`

Reply:
I thought we had mentioned we should move the Previews Controller out of the ark_manager all together to the commonwealth-vlr-engine or the front end DC 3.0 apps? Plus the controller is not currently in use as no endpoints can hit it